cmd/namespace/use.go

```diff
@@ -62,6 +62,10 @@ func Use(ctx context.Context) *cobra.Command {
 				return err
 			}
 			err = nsCmd.Use(ctx, namespace)
+			if err != nil {
```

Comment:
I think you need to remove this. That way we run `analytics.TrackNamespace(err == nil, len(args) > 0)` to track command errors